CHARTER.md

```diff
@@ -0,0 +1,61 @@
+# Charter for the Vega Projects
+
+## Scope and Goals
+
+This is the organizational charter for [Vega](https://github.com/vega/vega), [Vega-Lite](https://github.com/vega/vega-lite), [Vega-Altair](https://github.com/altair-viz/altair), [Vega-Embed](https://github.com/vega/vega-embed), [Vega-Datasets](https://github.com/vega/vega-datasets), [Vega-Themes](https://github.com/vega/vega-themes), [Vega-Tooltips](https://github.com/vega/vega-tooltip), [Vega-Editor](https://github.com/vega/editor), and [VL-Convert](https://github.com/vega/vl-convert). These projects are collectively referred to as the "Vega projects" in this document.
```

Comment:
```suggestion
This is the organizational charter for [Vega](https://github.com/vega/vega), [Vega-Lite](https://github.com/vega/vega-lite), [Vega-Altair](https://github.com/altair-viz/altair), [Vega-Embed](https://github.com/vega/vega-embed), [Vega-Datasets](https://github.com/vega/vega-datasets), [Vega-Themes](https://github.com/vega/vega-themes), [Vega-Tooltips](https://github.com/vega/vega-tooltip), [Vega-Editor](https://github.com/vega/editor), [VL-Convert](https://github.com/vega/vl-convert), [Altair-Tiles](https://github.com/altair-viz/altair_tiles), and [dash-vega-components](https://github.com/altair-viz/dash-vega-components). These projects are collectively referred to as the "Vega projects" in this document.
```
Added altair_tiles and dash-vega-components as projects as they are also under the current Vega-Altair governance. I'll create separate PRs in these projects to remove the `project-docs` and just refer to this repo.

Reply:
Good points. I added a note below about adding projects to the umbrella. Alternatively, we can just say all projects under the two github organizations are under the charter. Any reason we shouldn't do that?

Reply:
From my thinking it make sense to include all projects within the repo. 
One of the reason why I think it make sense is because there a few packages within the Vega-Altair repo that can potentially be deprecated/archived, but I think this requires a vote by the maintainers of the Altair-viz organization. 
Same is probably valid for some of the projects within the Vega organization I think.